library/src/test/java/com/getstream/sdk/chat/rest/ClientTest.java

```diff
@@ -0,0 +1,206 @@
+        channel.setName("test name");
+        Message updateMessage = new Message();
+        updateMessage.setText("test text");
+
+        client.updateChannel(channel, updateMessage, callback);
+
+        Mockito.verify(apiService).updateChannel(Mockito.eq(channelType),
+                Mockito.eq(channelId),
+                Mockito.eq(TEST_API_KEY),
+                Mockito.eq(TEST_CLIENT_ID),
+                Mockito.argThat(argument -> argument.getData().get("name").equals(channel.getName())
+                        && argument.getUpdateMessage().getText().equals(updateMessage.getText())));
+        Mockito.verify(callback).onSuccess(response);
+    }
+
+    @Test
+    void deleteChannelSuccessTest() {
+        ChannelResponse response = new ChannelResponse("100", new Channel());
+        ChannelCallback callback = Mockito.mock(ChannelCallback.class);
+
+        Mockito.when(apiService.deleteChannel(Mockito.any(), Mockito.any(), Mockito.any(), Mockito.any()))
+                .thenReturn(CallFake.buildSuccess(response));
+
+        String channelType = "testType";
+        String channelId = "testId";
+
+        client.deleteChannel(new Channel(client, channelType, channelId), callback);
+
+        Mockito.verify(apiService).deleteChannel(channelType, channelId, TEST_API_KEY, TEST_CLIENT_ID);
+        Mockito.verify(callback).onSuccess(response);
+    }
+
+    @Test
+    void banUserSuccessTest() {
+        CompletableResponse response = new CompletableResponse();
+        CompletableCallback callback = Mockito.mock(CompletableCallback.class);
+
+        Mockito.when(apiService.banUser(Mockito.any(), Mockito.any(), Mockito.any()))
+                .thenReturn(CallFake.buildSuccess(response));
+
+        String channelType = "testType";
+        String channelId = "testId";
+        String targetUserId = "testUserId";
+        String reason = "testReason";
+        int timeout = 100;
+
+        client.banUser(targetUserId, new Channel(client, channelType, channelId),
+                reason, timeout, callback);
+
+        Mockito.verify(apiService).banUser(Mockito.eq(TEST_API_KEY), Mockito.eq(TEST_CLIENT_ID),
+                Mockito.argThat(argument ->
+                        argument.getChannelType().equals(channelType) &&
+                                argument.getChannelId().equals(channelId) &&
+                                argument.getReason().equals(reason) &&
+                                argument.getTimeout() == timeout &&
+                                argument.getTargetUserId().equals(targetUserId)
+                )
+        );
+        Mockito.verify(callback).onSuccess(response);
+    }
+
+    @Test
+    void unBanUserSuccessTest() {
+        CompletableResponse response = new CompletableResponse();
+        CompletableCallback callback = Mockito.mock(CompletableCallback.class);
+
+        Mockito.when(apiService.unBanUser(Mockito.any(), Mockito.any(), Mockito.any(),
```

Comment:
Minor readability improvement would be just if you import `Mockito.*`. Otherwise it's 80 repeated `Mockito` :)